### ISSUE TITLE
Vaulting flow for Paypal not working (1.32.5)

### ISSUE THREAD

**Issue by Josefnm** [primer-io/primer-sdk-ios#360]
On completion when returning to app, i get this error. I looked a bit at the code and it seems like for the billing agreement session there is no orderId set in "PayPalTokenizationViewModel", but "generatePaypalPaymentInstrument()" is called which uses it.

```
{ 
     recoverySuggestion: 'Check if value nil is valid for key orderId',
     description: '[invalid-value] Invalid value \'nil\' for key \'orderId\'',
     errorId: 'PrimerSDK.PrimerError' 
} 
```

**Comment by xevious78** [comment]
Hey @Josefnm! 👋 We'll take a look into it shortly!
Could you tell us more about the setup that leads to this error?

**Comment by Josefnm** [comment]
Anything specific you want to know?

Xcode version: 13.2.1
Device simulator: iphone 12, ios 15.2
React Native: 0.66.4
React native primer sdk: 0.15.4 (but i have locally patched some bugs, so you can't reproduce it on your end using it) 

I am trying to vault a paypal payment method. It's set up on paypal, on your dashboard and on our backend with the newest version of your api, and I can successfully vault Paypal with android. 

client session token example:
```
{
    orderId: abc123,
    currencyCode: 'SEK',
    amount: 1,
    order: {
      countryCode: 'SE'
    }
  }
  ```

I'm using showPaymentMethod like this:
`showPaymentMethod(PaymentMethodConfigType.payPal, withIntent: PrimerSessionIntent.vault, on: UIViewController(), with: clientToken)`

After going through the paypal process and being redirected back to the app, an onError listener prints the error.

**Comment by xevious78** [comment]
Thanks for all of that, super helpful!
We were able to find the root cause the issue, we're working on a fix: #373

**Comment by EvansPie** [comment]
Hey @Josefnm , this was fixed on release 1.35.0. 

Also RN 0.15.8 is dependent upon iOS 1.35.0, so it should be fixed there as well.

Thanks for reporting the issue!
